composer.json

```diff
@@ -53,6 +54,11 @@
             "build/modules/contrib/{$name}": ["type:drupal-module"],
             "build/themes/contrib/{$name}": ["type:drupal-theme"]
         },
+        "patches": {
+            "drupal/core": {
+                "https://www.drupal.org/project/drupal/issues/3158130": "https://www.drupal.org/files/issues/2022-12-11/3158130-26.patch"
```

Comment:
Format for patches is
`"<issue title> @see <issue link>": "<patch url>"`.

Reply:
Fixed